contrib: update macOS SDK to Xcode-26.1.1-17B100

fanquake opened the pull request:
Updates the macOS SDK used for Guix builds to `Xcode-26.1.1-17B100`. 
Closes #34034.

fanquake committed:
contrib: update macOS SDK to Xcode-26.1.1-17B100

maflcko commented:
Forgot to bump the CI, according to the touched files of the last bump: `git show 8ea45e626e5a0482ee11d4376f961d8126ce7c7b --stat`?

fanquake commented:
Yea. However no point until the SDK is uploaded, and no point doing that until we've got determinism.

fanquake commented on contrib/macdeploy/README.md:
```diff
@@ -15,14 +15,14 @@ When complete, it will have produced `Bitcoin-Core.zip`.
 
 Our macOS SDK can be extracted from
-[Xcode_15.xip](https://download.developer.apple.com/Developer_Tools/Xcode_15/Xcode_15.xip).
+[Xcode_26.1.1_Apple_silicon.xip](https://download.developer.apple.com/Developer_Tools/Xcode_26.1.1/Xcode_26.1.1_Apple_silicon.xip).
```
> This implies dropping x86_64-apple-darwin?

No, this isn't dropping support for any build targets. IIrc this is the version of Xcode that only runs on Apple Silicon, which I chose because the download is smaller, and this doesn't make any difference to the extracted SDK.

theuni commented:
Concept ACK.

hebasto commented:
The CI failure does not appear to be specific to Qt.

For instance, `gmake -C depends zeromq HOST=arm64-apple-darwin` also fails.

fanquake committed:
depends: use -Xclang -fno-cxx-modules in macOS cross build

Newer versions of the macOS SDK, have introduced code like:
```cpp
if defined(__has_feature) && __has_feature(modules)
define USE_CLANG_TYPES 1
else
define USE_CLANG_TYPES 0
endif

if USE_CLANG_TYPES
include <sys/_types/_ptrdiff_t.h>
include <sys/_types/_size_t.h>
include <sys/_types/_va_list.h>
include <sys/_types/_wchar_t.h>
endif
```

which is currently causing compile failures due to undeclared types,
which manifest when C++ modules are enabled. Note that the usage of
"modules" in LLVM is can be a bit ambiguous, see:
https://github.com/llvm/llvm-project/issues/55891.

For now, explcitly disable cxx modules using `-fno-cxx-modules`. This
resolves the include/compilation issues.

Related discussion:
https://gcc.gnu.org/bugzilla/show_bug.cgi?id=116827
https://github.com/llvm/llvm-project/pull/150349
https://github.com/llvm/llvm-project/issues/57432

fanquake commented:
Guix Build (aarch64):
```bash
fb4b2da29ddeb09d319e81e001a49cfd7dafc4c6243eb7183e43187e328f76c8  guix-build-a89e1618dd8c/output/aarch64-linux-gnu/SHA256SUMS.part
7616912e8f4e8f74e50ba408715af9751c61924838cf2f7fa497202e66fed966  guix-build-a89e1618dd8c/output/aarch64-linux-gnu/bitcoin-a89e1618dd8c-aarch64-linux-gnu-debug.tar.gz
71c732be56e71697eaf6a68703d29d0361896a667beeb734fcde749c979f63f6  guix-build-a89e1618dd8c/output/aarch64-linux-gnu/bitcoin-a89e1618dd8c-aarch64-linux-gnu.tar.gz
b05a1a96e4cb20d525fc8f07cc8c95fd378df70b58fe398dcc8741c69e031a20  guix-build-a89e1618dd8c/output/arm-linux-gnueabihf/SHA256SUMS.part
0427df30912dbfddd85e399aea432e965cd2bc29c10e15b6365a0570248246bb  guix-build-a89e1618dd8c/output/arm-linux-gnueabihf/bitcoin-a89e1618dd8c-arm-linux-gnueabihf-debug.tar.gz
e5fc16a4b9c88fccba5fb34bbfb2170333b405d4f8bd90468aec004f0ec41c1c  guix-build-a89e1618dd8c/output/arm-linux-gnueabihf/bitcoin-a89e1618dd8c-arm-linux-gnueabihf.tar.gz
16f201523516c88722b86c72db0e0134bee9e2a4e2347d87fdae38c10198f216  guix-build-a89e1618dd8c/output/arm64-apple-darwin/SHA256SUMS.part
46a6eab83fdc2901d7d4d7f8179b42f6dd67e7bddf96ea8c5b345f71ee08faee  guix-build-a89e1618dd8c/output/arm64-apple-darwin/bitcoin-a89e1618dd8c-arm64-apple-darwin-codesigning.tar.gz
636b69532f5f07a7edc1c554a6d0dd58f77a2c55b4db2e5ce215addf2aedb571  guix-build-a89e1618dd8c/output/arm64-apple-darwin/bitcoin-a89e1618dd8c-arm64-apple-darwin-unsigned.tar.gz
ef6530d7a83856c6a81ef51fd590cb3300b66125ac95b3c0f09f43899f0c6baf  guix-build-a89e1618dd8c/output/arm64-apple-darwin/bitcoin-a89e1618dd8c-arm64-apple-darwin-unsigned.zip
a03854224336f108b6c26d4aa17879d602b048d78f0085c7adce46348b082793  guix-build-a89e1618dd8c/output/dist-archive/bitcoin-a89e1618dd8c.tar.gz
72bcd95a037e743513583a455658f257e4e2c8a0cf5f32952d63f4a7ed730fa5  guix-build-a89e1618dd8c/output/powerpc64-linux-gnu/SHA256SUMS.part
f9bf74b887ed70863248f563516aadfa768b4a323e5ccde28ff79e301c58bd3c  guix-build-a89e1618dd8c/output/powerpc64-linux-gnu/bitcoin-a89e1618dd8c-powerpc64-linux-gnu-debug.tar.gz
2a98c4c7404495c4b8d6a6f3efdbd80e0e3c9f97183603667e2dc002cf8ba61c  guix-build-a89e1618dd8c/output/powerpc64-linux-gnu/bitcoin-a89e1618dd8c-powerpc64-linux-gnu.tar.gz
28fd83a61cc4e14533d8e9a4ea7ce47a9b5614c007881f28ef508586015db73e  guix-build-a89e1618dd8c/output/riscv64-linux-gnu/SHA256SUMS.part
6ddb75ca17ea2ad51d208ba1745dc6c900bd9faabb4afa9d7dabc7e2c20f1a3c  guix-build-a89e1618dd8c/output/riscv64-linux-gnu/bitcoin-a89e1618dd8c-riscv64-linux-gnu-debug.tar.gz
b9ed71eca9322c33e94b3f27532074a80675b3a50e3841d60f244d715ab235dd  guix-build-a89e1618dd8c/output/riscv64-linux-gnu/bitcoin-a89e1618dd8c-riscv64-linux-gnu.tar.gz
196f5e1ed9adb2cf98b7084b40bacb9fae4e91e4ec4aeb35acbb9ea679bc54d6  guix-build-a89e1618dd8c/output/x86_64-apple-darwin/SHA256SUMS.part
90e59b5819a4fb85c98524497c9dd7de0a4646bad9bbf15b40da6e8ee49edb47  guix-build-a89e1618dd8c/output/x86_64-apple-darwin/bitcoin-a89e1618dd8c-x86_64-apple-darwin-codesigning.tar.gz
cc233a12b661c313cc737d115836334c4d245e13ffd28c4217b99d2cd4485625  guix-build-a89e1618dd8c/output/x86_64-apple-darwin/bitcoin-a89e1618dd8c-x86_64-apple-darwin-unsigned.tar.gz
c793968c65dc071477f1bc458671c1fc37984472b976d7021375b8eaf3dd82b1  guix-build-a89e1618dd8c/output/x86_64-apple-darwin/bitcoin-a89e1618dd8c-x86_64-apple-darwin-unsigned.zip
730987fc35ea6cd5f02490ed3f854824b9f9ed925778653cdfc14abfe374d166  guix-build-a89e1618dd8c/output/x86_64-linux-gnu/SHA256SUMS.part
8f16be7ac86cf26e3ef45fd6e5a642bf9e472327078dcbb494a0c5dc760101ec  guix-build-a89e1618dd8c/output/x86_64-linux-gnu/bitcoin-a89e1618dd8c-x86_64-linux-gnu-debug.tar.gz
9bb2a525edf2e313a56dbb2a9dbc1031f44076057e6e8afd16de67490f638eef  guix-build-a89e1618dd8c/output/x86_64-linux-gnu/bitcoin-a89e1618dd8c-x86_64-linux-gnu.tar.gz
40f8fb0dc9a02504d7265dc7772a4b465836b224a8152cdef6e8cb08d076dcd7  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/SHA256SUMS.part
6ff4fb1d1289ca41396bc229859010533475e6665f0e5f6250b8ee6d160ab191  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/bitcoin-a89e1618dd8c-win64-codesigning.tar.gz
a47a4c885ba2cca9f31e9f01cadd57b880e0a90d055c40ec33b66a6109a1fe1f  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/bitcoin-a89e1618dd8c-win64-debug.zip
c8a30bb669852210defec234bc653b82c61047e480ee09e81ead38c6f4f75ee8  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/bitcoin-a89e1618dd8c-win64-setup-unsigned.exe
220948977574da4eb4f8ca692a12b1f3918b97503c6b6e90d5875a51c48ea874  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/bitcoin-a89e1618dd8c-win64-unsigned.zip
```

hebasto reviewed (commented):
My Guix build:
```
x86_64
fb4b2da29ddeb09d319e81e001a49cfd7dafc4c6243eb7183e43187e328f76c8  guix-build-a89e1618dd8c/output/aarch64-linux-gnu/SHA256SUMS.part
7616912e8f4e8f74e50ba408715af9751c61924838cf2f7fa497202e66fed966  guix-build-a89e1618dd8c/output/aarch64-linux-gnu/bitcoin-a89e1618dd8c-aarch64-linux-gnu-debug.tar.gz
71c732be56e71697eaf6a68703d29d0361896a667beeb734fcde749c979f63f6  guix-build-a89e1618dd8c/output/aarch64-linux-gnu/bitcoin-a89e1618dd8c-aarch64-linux-gnu.tar.gz
b05a1a96e4cb20d525fc8f07cc8c95fd378df70b58fe398dcc8741c69e031a20  guix-build-a89e1618dd8c/output/arm-linux-gnueabihf/SHA256SUMS.part
0427df30912dbfddd85e399aea432e965cd2bc29c10e15b6365a0570248246bb  guix-build-a89e1618dd8c/output/arm-linux-gnueabihf/bitcoin-a89e1618dd8c-arm-linux-gnueabihf-debug.tar.gz
e5fc16a4b9c88fccba5fb34bbfb2170333b405d4f8bd90468aec004f0ec41c1c  guix-build-a89e1618dd8c/output/arm-linux-gnueabihf/bitcoin-a89e1618dd8c-arm-linux-gnueabihf.tar.gz
16f201523516c88722b86c72db0e0134bee9e2a4e2347d87fdae38c10198f216  guix-build-a89e1618dd8c/output/arm64-apple-darwin/SHA256SUMS.part
46a6eab83fdc2901d7d4d7f8179b42f6dd67e7bddf96ea8c5b345f71ee08faee  guix-build-a89e1618dd8c/output/arm64-apple-darwin/bitcoin-a89e1618dd8c-arm64-apple-darwin-codesigning.tar.gz
636b69532f5f07a7edc1c554a6d0dd58f77a2c55b4db2e5ce215addf2aedb571  guix-build-a89e1618dd8c/output/arm64-apple-darwin/bitcoin-a89e1618dd8c-arm64-apple-darwin-unsigned.tar.gz
ef6530d7a83856c6a81ef51fd590cb3300b66125ac95b3c0f09f43899f0c6baf  guix-build-a89e1618dd8c/output/arm64-apple-darwin/bitcoin-a89e1618dd8c-arm64-apple-darwin-unsigned.zip
a03854224336f108b6c26d4aa17879d602b048d78f0085c7adce46348b082793  guix-build-a89e1618dd8c/output/dist-archive/bitcoin-a89e1618dd8c.tar.gz
72bcd95a037e743513583a455658f257e4e2c8a0cf5f32952d63f4a7ed730fa5  guix-build-a89e1618dd8c/output/powerpc64-linux-gnu/SHA256SUMS.part
f9bf74b887ed70863248f563516aadfa768b4a323e5ccde28ff79e301c58bd3c  guix-build-a89e1618dd8c/output/powerpc64-linux-gnu/bitcoin-a89e1618dd8c-powerpc64-linux-gnu-debug.tar.gz
2a98c4c7404495c4b8d6a6f3efdbd80e0e3c9f97183603667e2dc002cf8ba61c  guix-build-a89e1618dd8c/output/powerpc64-linux-gnu/bitcoin-a89e1618dd8c-powerpc64-linux-gnu.tar.gz
28fd83a61cc4e14533d8e9a4ea7ce47a9b5614c007881f28ef508586015db73e  guix-build-a89e1618dd8c/output/riscv64-linux-gnu/SHA256SUMS.part
6ddb75ca17ea2ad51d208ba1745dc6c900bd9faabb4afa9d7dabc7e2c20f1a3c  guix-build-a89e1618dd8c/output/riscv64-linux-gnu/bitcoin-a89e1618dd8c-riscv64-linux-gnu-debug.tar.gz
b9ed71eca9322c33e94b3f27532074a80675b3a50e3841d60f244d715ab235dd  guix-build-a89e1618dd8c/output/riscv64-linux-gnu/bitcoin-a89e1618dd8c-riscv64-linux-gnu.tar.gz
196f5e1ed9adb2cf98b7084b40bacb9fae4e91e4ec4aeb35acbb9ea679bc54d6  guix-build-a89e1618dd8c/output/x86_64-apple-darwin/SHA256SUMS.part
90e59b5819a4fb85c98524497c9dd7de0a4646bad9bbf15b40da6e8ee49edb47  guix-build-a89e1618dd8c/output/x86_64-apple-darwin/bitcoin-a89e1618dd8c-x86_64-apple-darwin-codesigning.tar.gz
cc233a12b661c313cc737d115836334c4d245e13ffd28c4217b99d2cd4485625  guix-build-a89e1618dd8c/output/x86_64-apple-darwin/bitcoin-a89e1618dd8c-x86_64-apple-darwin-unsigned.tar.gz
c793968c65dc071477f1bc458671c1fc37984472b976d7021375b8eaf3dd82b1  guix-build-a89e1618dd8c/output/x86_64-apple-darwin/bitcoin-a89e1618dd8c-x86_64-apple-darwin-unsigned.zip
730987fc35ea6cd5f02490ed3f854824b9f9ed925778653cdfc14abfe374d166  guix-build-a89e1618dd8c/output/x86_64-linux-gnu/SHA256SUMS.part
8f16be7ac86cf26e3ef45fd6e5a642bf9e472327078dcbb494a0c5dc760101ec  guix-build-a89e1618dd8c/output/x86_64-linux-gnu/bitcoin-a89e1618dd8c-x86_64-linux-gnu-debug.tar.gz
9bb2a525edf2e313a56dbb2a9dbc1031f44076057e6e8afd16de67490f638eef  guix-build-a89e1618dd8c/output/x86_64-linux-gnu/bitcoin-a89e1618dd8c-x86_64-linux-gnu.tar.gz
40f8fb0dc9a02504d7265dc7772a4b465836b224a8152cdef6e8cb08d076dcd7  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/SHA256SUMS.part
6ff4fb1d1289ca41396bc229859010533475e6665f0e5f6250b8ee6d160ab191  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/bitcoin-a89e1618dd8c-win64-codesigning.tar.gz
a47a4c885ba2cca9f31e9f01cadd57b880e0a90d055c40ec33b66a6109a1fe1f  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/bitcoin-a89e1618dd8c-win64-debug.zip
c8a30bb669852210defec234bc653b82c61047e480ee09e81ead38c6f4f75ee8  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/bitcoin-a89e1618dd8c-win64-setup-unsigned.exe
220948977574da4eb4f8ca692a12b1f3918b97503c6b6e90d5875a51c48ea874  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/bitcoin-a89e1618dd8c-win64-unsigned.zip
```

hebasto reviewed (commented):
Tested my Guix binaries on macOS Tahoe 26.2 (Apple Silicon) after self-signing.

No issues.

Sjors commented:
x86_64 (running guix 1.5.0):

```
fb4b2da29ddeb09d319e81e001a49cfd7dafc4c6243eb7183e43187e328f76c8  guix-build-a89e1618dd8c/output/aarch64-linux-gnu/SHA256SUMS.part
7616912e8f4e8f74e50ba408715af9751c61924838cf2f7fa497202e66fed966  guix-build-a89e1618dd8c/output/aarch64-linux-gnu/bitcoin-a89e1618dd8c-aarch64-linux-gnu-debug.tar.gz
71c732be56e71697eaf6a68703d29d0361896a667beeb734fcde749c979f63f6  guix-build-a89e1618dd8c/output/aarch64-linux-gnu/bitcoin-a89e1618dd8c-aarch64-linux-gnu.tar.gz
b05a1a96e4cb20d525fc8f07cc8c95fd378df70b58fe398dcc8741c69e031a20  guix-build-a89e1618dd8c/output/arm-linux-gnueabihf/SHA256SUMS.part
0427df30912dbfddd85e399aea432e965cd2bc29c10e15b6365a0570248246bb  guix-build-a89e1618dd8c/output/arm-linux-gnueabihf/bitcoin-a89e1618dd8c-arm-linux-gnueabihf-debug.tar.gz
e5fc16a4b9c88fccba5fb34bbfb2170333b405d4f8bd90468aec004f0ec41c1c  guix-build-a89e1618dd8c/output/arm-linux-gnueabihf/bitcoin-a89e1618dd8c-arm-linux-gnueabihf.tar.gz
16f201523516c88722b86c72db0e0134bee9e2a4e2347d87fdae38c10198f216  guix-build-a89e1618dd8c/output/arm64-apple-darwin/SHA256SUMS.part
46a6eab83fdc2901d7d4d7f8179b42f6dd67e7bddf96ea8c5b345f71ee08faee  guix-build-a89e1618dd8c/output/arm64-apple-darwin/bitcoin-a89e1618dd8c-arm64-apple-darwin-codesigning.tar.gz
636b69532f5f07a7edc1c554a6d0dd58f77a2c55b4db2e5ce215addf2aedb571  guix-build-a89e1618dd8c/output/arm64-apple-darwin/bitcoin-a89e1618dd8c-arm64-apple-darwin-unsigned.tar.gz
ef6530d7a83856c6a81ef51fd590cb3300b66125ac95b3c0f09f43899f0c6baf  guix-build-a89e1618dd8c/output/arm64-apple-darwin/bitcoin-a89e1618dd8c-arm64-apple-darwin-unsigned.zip
a03854224336f108b6c26d4aa17879d602b048d78f0085c7adce46348b082793  guix-build-a89e1618dd8c/output/dist-archive/bitcoin-a89e1618dd8c.tar.gz
72bcd95a037e743513583a455658f257e4e2c8a0cf5f32952d63f4a7ed730fa5  guix-build-a89e1618dd8c/output/powerpc64-linux-gnu/SHA256SUMS.part
f9bf74b887ed70863248f563516aadfa768b4a323e5ccde28ff79e301c58bd3c  guix-build-a89e1618dd8c/output/powerpc64-linux-gnu/bitcoin-a89e1618dd8c-powerpc64-linux-gnu-debug.tar.gz
2a98c4c7404495c4b8d6a6f3efdbd80e0e3c9f97183603667e2dc002cf8ba61c  guix-build-a89e1618dd8c/output/powerpc64-linux-gnu/bitcoin-a89e1618dd8c-powerpc64-linux-gnu.tar.gz
28fd83a61cc4e14533d8e9a4ea7ce47a9b5614c007881f28ef508586015db73e  guix-build-a89e1618dd8c/output/riscv64-linux-gnu/SHA256SUMS.part
6ddb75ca17ea2ad51d208ba1745dc6c900bd9faabb4afa9d7dabc7e2c20f1a3c  guix-build-a89e1618dd8c/output/riscv64-linux-gnu/bitcoin-a89e1618dd8c-riscv64-linux-gnu-debug.tar.gz
b9ed71eca9322c33e94b3f27532074a80675b3a50e3841d60f244d715ab235dd  guix-build-a89e1618dd8c/output/riscv64-linux-gnu/bitcoin-a89e1618dd8c-riscv64-linux-gnu.tar.gz
196f5e1ed9adb2cf98b7084b40bacb9fae4e91e4ec4aeb35acbb9ea679bc54d6  guix-build-a89e1618dd8c/output/x86_64-apple-darwin/SHA256SUMS.part
90e59b5819a4fb85c98524497c9dd7de0a4646bad9bbf15b40da6e8ee49edb47  guix-build-a89e1618dd8c/output/x86_64-apple-darwin/bitcoin-a89e1618dd8c-x86_64-apple-darwin-codesigning.tar.gz
cc233a12b661c313cc737d115836334c4d245e13ffd28c4217b99d2cd4485625  guix-build-a89e1618dd8c/output/x86_64-apple-darwin/bitcoin-a89e1618dd8c-x86_64-apple-darwin-unsigned.tar.gz
c793968c65dc071477f1bc458671c1fc37984472b976d7021375b8eaf3dd82b1  guix-build-a89e1618dd8c/output/x86_64-apple-darwin/bitcoin-a89e1618dd8c-x86_64-apple-darwin-unsigned.zip
730987fc35ea6cd5f02490ed3f854824b9f9ed925778653cdfc14abfe374d166  guix-build-a89e1618dd8c/output/x86_64-linux-gnu/SHA256SUMS.part
8f16be7ac86cf26e3ef45fd6e5a642bf9e472327078dcbb494a0c5dc760101ec  guix-build-a89e1618dd8c/output/x86_64-linux-gnu/bitcoin-a89e1618dd8c-x86_64-linux-gnu-debug.tar.gz
9bb2a525edf2e313a56dbb2a9dbc1031f44076057e6e8afd16de67490f638eef  guix-build-a89e1618dd8c/output/x86_64-linux-gnu/bitcoin-a89e1618dd8c-x86_64-linux-gnu.tar.gz
40f8fb0dc9a02504d7265dc7772a4b465836b224a8152cdef6e8cb08d076dcd7  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/SHA256SUMS.part
6ff4fb1d1289ca41396bc229859010533475e6665f0e5f6250b8ee6d160ab191  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/bitcoin-a89e1618dd8c-win64-codesigning.tar.gz
a47a4c885ba2cca9f31e9f01cadd57b880e0a90d055c40ec33b66a6109a1fe1f  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/bitcoin-a89e1618dd8c-win64-debug.zip
c8a30bb669852210defec234bc653b82c61047e480ee09e81ead38c6f4f75ee8  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/bitcoin-a89e1618dd8c-win64-setup-unsigned.exe
220948977574da4eb4f8ca692a12b1f3918b97503c6b6e90d5875a51c48ea874  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/bitcoin-a89e1618dd8c-win64-unsigned.zip
```

sedited reviewed (approved):
Guix builds:
```
aarch64
fb4b2da29ddeb09d319e81e001a49cfd7dafc4c6243eb7183e43187e328f76c8  guix-build-a89e1618dd8c/output/aarch64-linux-gnu/SHA256SUMS.part
7616912e8f4e8f74e50ba408715af9751c61924838cf2f7fa497202e66fed966  guix-build-a89e1618dd8c/output/aarch64-linux-gnu/bitcoin-a89e1618dd8c-aarch64-linux-gnu-debug.tar.gz
71c732be56e71697eaf6a68703d29d0361896a667beeb734fcde749c979f63f6  guix-build-a89e1618dd8c/output/aarch64-linux-gnu/bitcoin-a89e1618dd8c-aarch64-linux-gnu.tar.gz
b05a1a96e4cb20d525fc8f07cc8c95fd378df70b58fe398dcc8741c69e031a20  guix-build-a89e1618dd8c/output/arm-linux-gnueabihf/SHA256SUMS.part
0427df30912dbfddd85e399aea432e965cd2bc29c10e15b6365a0570248246bb  guix-build-a89e1618dd8c/output/arm-linux-gnueabihf/bitcoin-a89e1618dd8c-arm-linux-gnueabihf-debug.tar.gz
e5fc16a4b9c88fccba5fb34bbfb2170333b405d4f8bd90468aec004f0ec41c1c  guix-build-a89e1618dd8c/output/arm-linux-gnueabihf/bitcoin-a89e1618dd8c-arm-linux-gnueabihf.tar.gz
16f201523516c88722b86c72db0e0134bee9e2a4e2347d87fdae38c10198f216  guix-build-a89e1618dd8c/output/arm64-apple-darwin/SHA256SUMS.part
46a6eab83fdc2901d7d4d7f8179b42f6dd67e7bddf96ea8c5b345f71ee08faee  guix-build-a89e1618dd8c/output/arm64-apple-darwin/bitcoin-a89e1618dd8c-arm64-apple-darwin-codesigning.tar.gz
636b69532f5f07a7edc1c554a6d0dd58f77a2c55b4db2e5ce215addf2aedb571  guix-build-a89e1618dd8c/output/arm64-apple-darwin/bitcoin-a89e1618dd8c-arm64-apple-darwin-unsigned.tar.gz
ef6530d7a83856c6a81ef51fd590cb3300b66125ac95b3c0f09f43899f0c6baf  guix-build-a89e1618dd8c/output/arm64-apple-darwin/bitcoin-a89e1618dd8c-arm64-apple-darwin-unsigned.zip
a03854224336f108b6c26d4aa17879d602b048d78f0085c7adce46348b082793  guix-build-a89e1618dd8c/output/dist-archive/bitcoin-a89e1618dd8c.tar.gz
72bcd95a037e743513583a455658f257e4e2c8a0cf5f32952d63f4a7ed730fa5  guix-build-a89e1618dd8c/output/powerpc64-linux-gnu/SHA256SUMS.part
f9bf74b887ed70863248f563516aadfa768b4a323e5ccde28ff79e301c58bd3c  guix-build-a89e1618dd8c/output/powerpc64-linux-gnu/bitcoin-a89e1618dd8c-powerpc64-linux-gnu-debug.tar.gz
2a98c4c7404495c4b8d6a6f3efdbd80e0e3c9f97183603667e2dc002cf8ba61c  guix-build-a89e1618dd8c/output/powerpc64-linux-gnu/bitcoin-a89e1618dd8c-powerpc64-linux-gnu.tar.gz
28fd83a61cc4e14533d8e9a4ea7ce47a9b5614c007881f28ef508586015db73e  guix-build-a89e1618dd8c/output/riscv64-linux-gnu/SHA256SUMS.part
6ddb75ca17ea2ad51d208ba1745dc6c900bd9faabb4afa9d7dabc7e2c20f1a3c  guix-build-a89e1618dd8c/output/riscv64-linux-gnu/bitcoin-a89e1618dd8c-riscv64-linux-gnu-debug.tar.gz
b9ed71eca9322c33e94b3f27532074a80675b3a50e3841d60f244d715ab235dd  guix-build-a89e1618dd8c/output/riscv64-linux-gnu/bitcoin-a89e1618dd8c-riscv64-linux-gnu.tar.gz
196f5e1ed9adb2cf98b7084b40bacb9fae4e91e4ec4aeb35acbb9ea679bc54d6  guix-build-a89e1618dd8c/output/x86_64-apple-darwin/SHA256SUMS.part
90e59b5819a4fb85c98524497c9dd7de0a4646bad9bbf15b40da6e8ee49edb47  guix-build-a89e1618dd8c/output/x86_64-apple-darwin/bitcoin-a89e1618dd8c-x86_64-apple-darwin-codesigning.tar.gz
cc233a12b661c313cc737d115836334c4d245e13ffd28c4217b99d2cd4485625  guix-build-a89e1618dd8c/output/x86_64-apple-darwin/bitcoin-a89e1618dd8c-x86_64-apple-darwin-unsigned.tar.gz
c793968c65dc071477f1bc458671c1fc37984472b976d7021375b8eaf3dd82b1  guix-build-a89e1618dd8c/output/x86_64-apple-darwin/bitcoin-a89e1618dd8c-x86_64-apple-darwin-unsigned.zip
730987fc35ea6cd5f02490ed3f854824b9f9ed925778653cdfc14abfe374d166  guix-build-a89e1618dd8c/output/x86_64-linux-gnu/SHA256SUMS.part
8f16be7ac86cf26e3ef45fd6e5a642bf9e472327078dcbb494a0c5dc760101ec  guix-build-a89e1618dd8c/output/x86_64-linux-gnu/bitcoin-a89e1618dd8c-x86_64-linux-gnu-debug.tar.gz
9bb2a525edf2e313a56dbb2a9dbc1031f44076057e6e8afd16de67490f638eef  guix-build-a89e1618dd8c/output/x86_64-linux-gnu/bitcoin-a89e1618dd8c-x86_64-linux-gnu.tar.gz
40f8fb0dc9a02504d7265dc7772a4b465836b224a8152cdef6e8cb08d076dcd7  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/SHA256SUMS.part
6ff4fb1d1289ca41396bc229859010533475e6665f0e5f6250b8ee6d160ab191  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/bitcoin-a89e1618dd8c-win64-codesigning.tar.gz
a47a4c885ba2cca9f31e9f01cadd57b880e0a90d055c40ec33b66a6109a1fe1f  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/bitcoin-a89e1618dd8c-win64-debug.zip
c8a30bb669852210defec234bc653b82c61047e480ee09e81ead38c6f4f75ee8  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/bitcoin-a89e1618dd8c-win64-setup-unsigned.exe
220948977574da4eb4f8ca692a12b1f3918b97503c6b6e90d5875a51c48ea874  guix-build-a89e1618dd8c/output/x86_64-w64-mingw32/bitcoin-a89e1618dd8c-win64-unsigned.zip
```

Also did a non-guix cross-compile run. Didn't check if the binaries actually work.

ACK a89e1618dd8c2ae655093c947fc3f51a34aad1a0

Edit: I guess it would be good if someone tested the x86 bins?

janb84 reviewed (commented):
concept ACK a89e1618dd8c2ae655093c947fc3f51a34aad1a0

Followed guide all hashes match. Guix build works, got [matching hashes ](https://github.com/bitcoin/bitcoin/pull/34036#issuecomment-3829324664).
Build on Aarch64, Tested (limited) the bins on Linux X86-64 and Old x86-64 (intel) mac (MacOS 14.6.1).
